test/Microsoft.Azure.Devices.Edge.Test.Common/Platform.cs

```diff
@@ -1,27 +1,116 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace Microsoft.Azure.Devices.Edge.Test.Common
 {
-    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
     using System.Runtime.InteropServices;
+    using System.Security.Cryptography.X509Certificates;
     using System.Threading;
     using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
     using Microsoft.Azure.Devices.Edge.Util;
+    using Serilog;
 
-    public static class Platform
+    public class Platform
```

Comment:
```suggestion
    internal class Platform
```

Reply:
Since GenerateEdgeCertificatesAsync allows to pass command name and args, I think this method should not be called from outside, therefore I suggest to change this class to internal.

Reply:
It can't be internal because it's not in the same assembly as the tests. I did add back the `static` modifier though--that was an oversight.

Reply:
The tests need access to `Platform.Current`, while the library needs everything. I can probably split them apart and make the library-facing side internal. Stay tuned...

Reply:
Fixed. I made the shared cert-gen functions (the ones that take a command name and args) `protected`, and the platform-specific classes inherit `Common.Platform, IPlatform`.